instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/InstrumenterBuilder.java

```diff
@@ -26,6 +32,12 @@
  */
 public final class InstrumenterBuilder<REQUEST, RESPONSE> {
 
+  /** Instrumentation type suppression configuration property key. */
+  private static final boolean ENABLE_SPAN_SUPPRESSION_BY_TYPE =
+      Config.get()
+          .getBooleanProperty(
+              "otel.instrumentation.experimental.outgoing-span-suppression-by-type", false);
```

Comment:
Since this can be overridden through direct call to `enableInstrumentationTypeSuppression` I think we should use `default-` in the property name. Or maybe can remove that method

Reply:
removing `enableInstrumentationTypeSuppression` method sounds good 👍

Reply:
removing this method makes suppression untestable through instrumetner API. The method is package-private, though, so instrumentations won't be able to call it.